### PR TITLE
Issue #276: Correction of CustomConcatMethod

### PR DESCRIPTION
**Pull request by TNohaic**
Correction of CustomConcatMethod that is not always resulting in a well formatted JSON

### CODE REVIEW

File: metricshub-doc/src/site/markdown/develop/sources/http.md [hunk 1]
```diff
@@ -27,5 +27,6 @@ monitors:
             concatMethod: # onOf [ <enum>, <object> ] | possible values for <enum> : [ list, json_array, json_array_extended ]
             concatStart: # <string>
             concatEnd: # <string>
+              isJsonArray: # <boolean>
```
Comment (NassimBtk) [inline metricshub-doc/src/site/markdown/develop/sources/http.md:30]: indentation issue.

**Comment by NassimBtk** [comment]
I am good with this. @bertysentry? What do you think?

**Comment by bertysentry** [comment]
I wanted to understand the use case, and why the current JSON array concatenation options don't work.

**Comment by NassimBtk** [comment]
We are going to develop the AWK Source: https://github.com/sentrysoftware/metricshub/issues/316

**Pull request closed without merging**